code/etc/config.xml

```diff
@@ -49,18 +49,17 @@
     </adminhtml>
     <default>
         <meanbee_gmailactions_options>
-            <shipment_email>
-                <action_name>View Shipment</action_name>
-                <description>Open shipment in browser</description>
-            </shipment_email>
             <order_email>
-                <action_name>View Order</action_name>
-                <description>Open order in browser</description>
+                <enabled>1</enabled>
             </order_email>
+            <shipment_email>
+                <enabled>1</enabled>
+            </shipment_email>
             <creditmemo_email>
+                <enabled>1</enabled>
                 <action_name>View Credit Memo</action_name>
                 <description>Open credit memo in browser</description>
```

Comment:
@punkstar No.  Credit Memo is still using buttons, as there are no rich snippets for it.  These two are still being used.